app/models/reservation_scopes.rb

```diff
@@ -56,32 +42,29 @@ def self.included(base) # rubocop:disable MethodLength, AbcSize
       scope :ends_on_days, lambda { |start_date, end_date|
         where(due_date: start_date..end_date)
       }
-      scope :reserved_on_date, ->(date) { overlaps_with_date(date).finalized }
+      scope :reserved_on_date, lambda { |date|
+        overlaps_with_date(date).not_available
+      }
       scope :for_eq_model, lambda { |eq_model|
-        where(equipment_model_id: eq_model.id).finalized
+        where(equipment_model_id: eq_model.id)
       }
       scope :active_or_requested, lambda {
-        where('checked_in IS NULL and approval_status != ?', 'denied').recent
+        where(status: Reservation.statuses.values_at(
+          *%w(requested reserved checked_out)).compact)
       }
       scope :notes_unsent, ->() { where(notes_unsent: true) }
-      scope :requested, lambda {
-        where('start_date >= ? and approval_status = ?',
-              Time.zone.today, 'requested').recent
-      }
+
       scope :approved_requests, lambda {
-        where('approval_status = ?', 'approved').recent
-      }
-      scope :denied_requests, lambda {
-        where('approval_status = ?', 'denied').recent
+        where('flags & ? > 0', Reservation::FLAGS[:request]).reserved
       }
+
       scope :missed_requests, lambda {
-        where('approval_status = ? and start_date < ?', 'requested',
-              Time.zone.today).recent
+        where('start_date < ?', Time.zone.today).requested
```

Comment:
Do we want to make an enum status for this as well? I feel like it's similar to the regular "missed" scope.